src/integration/java/io/pinecone/integration/controlPlane/pod/CollectionTest.java

```diff
@@ -20,65 +20,81 @@
 import static io.pinecone.helpers.IndexManager.waitUntilIndexIsReady;
 import static io.pinecone.helpers.IndexManager.createCollection;
 import static io.pinecone.helpers.BuildUpsertRequest.*;
+import static io.pinecone.helpers.AssertRetry.assertWithRetry;
 import static org.junit.jupiter.api.Assertions.*;
 
 @Disabled("Disable the entire class")
 public class CollectionTest {
-
-    private static PineconeControlPlaneClient controlPlaneClient;
     private static final String indexName = RandomStringBuilder.build("collection-test", 8);
+    private static final String collectionName = RandomStringBuilder.build("collection-test", 8);
```

Comment:
Previously, each test in this file created a new collection, which was unnecessary and time consuming. We can create a single collection in our setup and share across both tests to increase speed.

Reply:
Great, thanks so much for adding this change!